packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -446,6 +470,24 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
         context,
         _isChildExpanded(index),
       );
+      final Widget expansionIndicator = child.expansionIndicator != null
+          ? InkResponse(
+        onTap: !child.canTapOnHeader
+            ? () => _handlePressed(_isChildExpanded(index), index)
```

Comment:
Do not use arrow functions when not explicitly returning a value, since it could be confusing to first-time readers of this code. Instead, use regular function expressions.